app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -42,6 +43,12 @@ def host_title_actions(*args)
         button_group(web_console_button(*args)))
       super(*args)
     end
-  end
 
+    def user_job_actions
+      JobAction.authorized(:view_job_actions)
+               .where(user: User.current)
+               .order(:name)
+               .map { |a| [a.name, new_job_invocation_path(template_id: a.job_template.id)] }
```

Comment:
Maybe just `a.job_template_id` to prevent loading the entire template when only id is needed?